packages/components/package.json

```diff
@@ -158,7 +158,7 @@
     "rollup": "^1.0.0",
     "rollup-plugin-babel": "^4.0.0",
     "rollup-plugin-commonjs": "^9.0.0",
-    "rollup-plugin-filesize": "^6.0.0",
+    "rollup-plugin-filesize": "6.1.1",
```

Comment:
Was this intentional?

Reply:
Yeah, `rollup-plugin-filesize@>6.1.1` includes `brotli-size@4.0.0` which requires node 10.16, which breaks the build (Circle is set up on 10.15)

Reply:
Can totally change whatever, but pinning to `6.1.1.` seemed to be the most compatible fix